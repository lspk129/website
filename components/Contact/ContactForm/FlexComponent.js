import styled from 'styled-components';

const FlexComponent = styled.div`
  display: flex;
  justify-content: ${props => (props.between ? 'space-between' : '')};
  flex-wrap: wrap;
  align-items: center;
`;

export default FlexComponent;
