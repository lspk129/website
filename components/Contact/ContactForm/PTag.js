import styled from 'styled-components';

const PTag = styled.p`
  padding-left: 20px;
`;

export default PTag;
