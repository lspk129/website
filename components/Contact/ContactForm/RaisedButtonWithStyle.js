import RaisedButton from 'material-ui/RaisedButton';
import styled from 'styled-components';

const RaisedButtonWithStyle = styled(RaisedButton)`
  margin-top: 30px;
  justify-self: start;
`;

export default RaisedButtonWithStyle;
