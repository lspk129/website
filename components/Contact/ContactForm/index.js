import styled from 'styled-components';

const ContactForm = styled.form``;

export default ContactForm;
