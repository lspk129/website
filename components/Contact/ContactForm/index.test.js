import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import ContactForm from './';

describe('ContactForm', () => {
  const shallowWrapper = shallow(<ContactForm />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<ContactForm />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders form element', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper).toHaveTagName('form');
  });
});
