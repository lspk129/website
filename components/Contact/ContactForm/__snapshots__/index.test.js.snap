// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactForm renders correctly 1`] = `
<form
  className="sc-bdVaJa bDWFJH"
/>
`;
